lib/matplotlib/text.py

```diff
@@ -601,6 +715,10 @@ def draw(self, renderer):
 
         renderer.open_group('text', self.get_gid())
 
+        if self.get_wrap():
```

Comment:
This maybe should be done with a context manager to make sure it is always restored correctly?